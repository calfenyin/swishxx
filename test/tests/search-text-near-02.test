search | | -i text.index | free near access | 0
