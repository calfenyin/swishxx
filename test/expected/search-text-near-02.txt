# results: 1
100 ./Gutenberg_License.txt 17308 Gutenberg_License.txt
